src/main/java/com/checkmarx/ast/wrapper/CxWrapper.java

```diff
@@ -336,7 +352,7 @@ public KicsRealtimeResults kicsRealtimeScan(@NonNull String fileSources, String
         List<String> arguments = new ArrayList<>();
         arguments.add(CxConstants.CMD_SCAN);
         arguments.add(CxConstants.SUB_CMD_KICS_REALTIME);
-        arguments.add(CxConstants.FILE_SOURCES);
+        arguments.add(CxConstants.FILE);
```

Comment:
what is this change?

Reply:
Changing an existing flag to a more indicative name. Previously it was 
FILE_SOURCES = --file
so I simply changed the constant name to what it actually is.